tests/data/Makefile.inc

```diff
@@ -58,7 +58,7 @@ test307 test308 test309 test310 test311 test312 test313 test314 test315 \
 test316 test317 test318 test319 test320 test321 test322 test323 test324 \
 test325 test326 test327 test328 test329 test330 test331 test332 test333 \
 test334 test335 test336 test337 test338 test339 test340 test341 test342 \
-test343 test344 test345 \
+test343 test344 test345 test347 test348 \
 test350 test351 test352 test353 test354 test355 test356 test357 test358 \
```

Comment:
You skipped test346, which is Okay. We just normally leave a gap there then to make it easier to spot.

Reply:
I'll move the files to another range... I was a bit conservative.